### DISCUSSION TITLE
Cloud Infrastructure Hackday 2024-11-19

### DISCUSSION THREAD

**Discussion by ateucher** [Openscapes/openscapes.cloud#12] [Hack days]
Reporting out on Cloud Infrastructure hack days.

Please create a new comment (at the very bottom, use the text box with a button that says "comment", not "reply") to write about what you did today!

Please include links to GitHub issues or pull requests relevant to your work, and use those issues or pull requests as the "source of truth" for documenting the work; this makes future readers jobs easier :) It's OK to duplicate some information while writing a summary here!

Use the "reply" feature to have a discussion under any given comment.

**Comment by jules32** [comment]
Julie updated some language for purpose & audience on the homepage for openscapes.cloud: 

> Openscapes supports partners leading workshops to provide friendly first experiences with cloud computing, and this website is both for users to access JupyterHubs and for workshop leads to administer workshops.

PR: https://github.com/Openscapes/openscapes.cloud/pull/13/files

**Comment by itcarroll** [comment]
Took 20 minutes to flesh out #7 with some notes.

**Comment by ateucher** [comment]
Jenny gave an overview of the new cost monitoring dashboards in Grafana, that 2i2c are rolling out across AWS hubs soon. These dashboards break costs by hubs (if there are multiple) and by component (compute, home storage, backup, networking, object storage, and fixed). Dashboards can be shared/exported for users who don't have a Grafana account.

Jenny shared [new docs](https://docs.2i2c.org/admin/howto/monitoring/prometheus-access/) in the 2i2c documentation for accessing prometheus data sources outside Grafana.
